pipes/WDL/workflows/blastoff.wdl

```diff
@@ -0,0 +1,56 @@
+version 1.0
+
+import "../tasks/tasks_megablast.wdl" as tools
```

Comment:
Maybe change the input alias to something more descriptive like `metablast` rather than `tools` (& change usage below to match)